bin/condor_submit_dag

```diff
@@ -59,14 +59,16 @@ def main():
     bindir = os.path.abspath(os.path.dirname(__file__))
     os.environ["_condor_SEC_CREDENTIAL_STORER"] = f"{bindir}/condor_vault_storer"
 
-    parser = get_parser.get_submit_parser(add_condor_epilog=True)
+    parser = get_parser.get_submit_parser()
 
     arglist, cmd_args = parser.parse_known_args()
 
     # pull dag file out of args
     if len(cmd_args):
         f = cmd_args[-1]
         cmd_args = cmd_args[:-1]
+    else:
+        raise SystemExit("Filename argument is required.")
```

Comment:
This exception will not get caught by our general error handling (e.g. in jobsub_submit), since `SystemExit` doesn't inherit from `Exception` (it inherits from `BaseException`).  

1. Is that OK?
2. If not, would you rather change this exception class here (maybe a `RuntimeError`?) , or change all of the general handling clauses to catch `BaseException`?

Reply:
We could raise argparse.ArgumentError... But this is the mainline for the condor_submit_dag wrapper, and there isn't any exception handling upstream of it.

Reply:
....you're absolutely right. Please ignore this comment.